.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
+  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+  
+  If you're publishing a new extension to OpenVSX, please make sure the extension has an OSI-approved open source license. If you want to have an extension with a proprietary or non-approved license, please ask its maintainers to publish it.
+-->
+
+## What type of PR is this?
```

Comment:
I don't think it is important. I would like to bring attention both on issues and PR for new or broken extensions to the fact that extension should be removed from this repo and published directly to OpenVSX by the author.